COMMUNITY_BADGES.md

```diff
@@ -0,0 +1,67 @@
+# OPA Community Badges
+
+Hello OPA Community Members, we thank you for taking the time to be a part of the community and for all of the work you’ve put in. As thanks, the OPA community now offers badges to recognize your contributions to the community.
```

Comment:
Suggest breaking this into two sentences:
```
Hello OPA Community Members! We thank you ...
```

Reply:
Updated